test_conformance/vulkan/test_vulkan_api_consistency.cpp

```diff
@@ -323,27 +323,6 @@ int test_consistency_external_image(cl_device_id deviceID, cl_context _context,
     test_error(errNum, "Unable to create Image with Properties");
     image.reset();
 
-    // Passing image_format as NULL
-    image = clCreateImageWithProperties(context, extMemProperties.data(),
```

Comment:
This is not related to semaphore negative tests, but for image import negative testing and should be retained, right?

Reply:
These failure conditions are already described in the OpenCL base specification.  If coverage is missing, they should be added to one of the core tests, such as API or basic.  The exception is if cl_khr_external_memory has a special provision for image_format or image_desc that I am forgetting.